types/knockout/index.d.ts

```diff
@@ -139,24 +139,70 @@ interface KnockoutSubscribableStatic {
 }
 
 interface KnockoutSubscription {
+    /**
+     * Terminates a subscription 
+     */
     dispose(): void;
 }
 
 interface KnockoutSubscribable<T> extends KnockoutSubscribableFunctions<T> {
-    subscribe(callback: (newValue: T) => void, target: any, event: "beforeChange"): KnockoutSubscription;
+    /**
+     * Registers to be notified after the observable's value changes
+     * @param callback Function that is called whenever the notification happens
+     * @param target? Defines the value of 'this' in the callback function
+     * @param event? The name of the event to receive notification for
+     */
     subscribe(callback: (newValue: T) => void, target?: any, event?: "change"): KnockoutSubscription;
+    /**
+     * Registers to be notified before the observable's value changes
+     * @param callback Function that is called whenever the notification happens
+     * @param target Defines the value of 'this' in the callback function
+     * @param event The name of the event to receive notification for
+     */
+    subscribe(callback: (newValue: T) => void, target: any, event: "beforeChange"): KnockoutSubscription;
+    /**
+     * Registers to be notified when the observable's value changes
+     * @param callback Function that is called whenever the notification happens
+     * @param target Defines the value of 'this' in the callback function
+     * @param event The name of the event to receive notification for
+     */
     subscribe<TEvent>(callback: (newValue: TEvent) => void, target: any, event: string): KnockoutSubscription;
-
+    /**
+     * Customizes observables basic functionality
+     * @param requestedExtenders Name of the extender feature and it's value, e.g. { notify: 'always' }, { rateLimit: 50 }
+     */
     extend(requestedExtenders: { [key: string]: any; }): KnockoutSubscribable<T>;
+    /**
+    * Gets total number of subscriptors
```

Comment:
Nit: "subscriptions", or "subscribers", would probably be better than "subscribers".

Reply:
good eye!